users/urls_api.py

```diff
@@ -2,8 +2,8 @@
 from . import views_api
 
 urlpatterns = [
-    path('users/', views_api.UsersView.as_view()),
-    path('users/me/', views_api.UserDetailView.as_view()),
-    path('auth/login/', views_api.LoginView.as_view()),
-    path('auth/change-password/', views_api.ChangePasswordView.as_view()),
+    path('users/', views_api.UsersView.as_view(), name="api-create-user"),
+    path('users/me/', views_api.UserDetailView.as_view(), name="api-edit-user"),
```

Comment:
This is not an endpoint to give any user detail. That's why UserDetail is not suitable for a view name.

We can use `MyUserDetailView`.


name="api-edit-user" => Let's not put logic(edit,create,list) in the url name. Because a new action can be added to the endpoint. In that case the url name will be inconsistent.